docs/Override.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Symbol Override
+order: 211
+section: components
+---
+
+```javascript
+var overrides = symbolInstance.overrides
+```
+
+A [Symbol](https://www.sketchapp.com/docs/symbols/) override. This component is not exposed, it is only returns when accessing the `overrides` of a [Symbol Instance](#symbol-instance).
```

Comment:
As with the other PRs, the use of "it is only returns when" reads a little odd. Would either "it is only returned when" or "it only returns when" be correct?